opendbc/car/hyundai/values.py

```diff
@@ -527,6 +527,16 @@ class CAR(Platforms):
     CarSpecs(mass=1950, wheelbase=2.87, steerRatio=14.6),
     flags=HyundaiFlags.RADAR_SCC,
   )
+  # 2023 GV70 Electrified has Variable Gear Ratio (VGR) rack & pinion with rack mounted EPS.
+  # The overall gear/steering ratio is 13.27.
+  # https://www.genesis.com/content/dam/genesis-p2/au/assets/models/electrified-gv70/specs/Genesis_GV70_EV_August_2022_v0.1.pdf
+  GENESIS_GV70_ELECTRIFIED_1ST_GEN = HyundaiCanFDPlatformConfig(
+    [
+      HyundaiCarDocs("Genesis GV70 Electrified (with HDA II) 2023", "Highway Driving Assist II", car_parts=CarParts.common([CarHarness.hyundai_q])),
+    ],
+    CarSpecs(mass=2295, wheelbase=2.87, steerRatio=13.27),
```

Comment:
Adjusting steerRatio to online learned value, slight tweak to mass per spec from Edmunds.

```suggestion
    CarSpecs(mass=2260, wheelbase=2.87, steerRatio=17.1),
```

![image](https://github.com/user-attachments/assets/13d2802a-66be-4b01-b9e1-1120e78bb924)